src/io/flutter/view/FlutterView.java

```diff
@@ -135,7 +141,23 @@ void initToolWindow(ToolWindow window) {
       final AnAction sendFeedbackAction = new AnAction("Send Feedback", "Send Feedback", FlutterIcons.Feedback) {
         @Override
         public void actionPerformed(@NotNull AnActionEvent event) {
-          BrowserUtil.browse("https://goo.gl/WrMB43");
+          FlutterSdkVersion flutterSdkVersion = FlutterSdk.getFlutterSdk(myProject).getVersion();
+          String pluginVersion = PluginManager.getPlugin(FlutterUtils.getPluginId()).getVersion();
+          String ideVersion = String.format("%s %s", PlatformUtils.getPlatformPrefix(), ApplicationInfo.getInstance().getStrictVersion());
```

Comment:
add another field for the platform prefix instead of adding it to the ideVersion.

Reply:
Done.